src/main/resources/view/EditWindow.css

```diff
@@ -0,0 +1,23 @@
+#addContainer {
+    -fx-background-color: derive(#1d1d1d, 20%);
+}
+
+#nameLabel {
+    -fx-background-color: #6F686D;
+}
+
+#phoneLabel {
+    -fx-background-color: #6F686D;
+}
+
+#addressLabel {
+    -fx-background-color: #6F686D;
+}
+
+#emailLabel {
+    -fx-background-color: #6F686D;
+}
+
+#indexLabel {
+    -fx-background-color: #6F686D;
+}
```

Comment:
Other than index, identical to AddWindow.css. Can merge into window.css?

Reply:
Good point. Will make the necessary changes sometime soon!